java/Graql.java

```diff
@@ -81,6 +84,17 @@ public static BoundVariable parseVariable(String variable) {
         return parser.parseVariableEOF(variable);
     }
 
+    public static String parseLabel(String label) {
+        String parsedLabel;
+        try {
+            parsedLabel = parser.parseLabelEOF(label);
+        } catch (GraqlException e) {
+            throw GraqlException.of(ILLEGAL_CHAR_IN_LABEL.message(label));
+        }
+        if (!parsedLabel.equals(label)) throw GraqlException.of(PARSED_LABEL_DIFFERS_FROM_RAW.message(label, parsedLabel));
```

Comment:
Reproducible examples:
- `tx.concepts().putEntityType(" abc")`
- `tx.concepts().putEntityType("person#123")`

Reply:
Let's just use one error message you have above.